src/ec.rs

```diff
@@ -535,7 +560,15 @@ where
       }
     }
     self.symbol(s, &cdf[..nsymbs]);
-    Self::update_cdf(cdf, s);
+    if cfg!(target_feature = "sse2") {
+      if cdf.len() == 5 {
+        Self::update_cdf_4_sse2(cdf, s);
+      } else {
+        Self::update_cdf(cdf, s);
```

Comment:
Deduplicate this. You can `return Self::update_cdf_4_sse2(cdf, s);` or something else.